src/styles/index.ts

```diff
@@ -1782,7 +1782,7 @@ const styles = (theme: ThemeColors) =>
         },
 
         sidebarLinkActive: {
-            backgroundColor: theme.buttonHoveredBG,
+            backgroundColor: theme.overlay,
```

Comment:
Why are we using the overlay color here? That's not the correct color we should be using. For the active background color, we want to use product400 or the same color as our borders.